src/TestingUtils/Microsoft.AspNetCore.Testing/src/HttpClientSlim.cs

```diff
@@ -94,6 +100,32 @@ private static async Task<string> ReadResponse(Stream stream)
             }
         }
 
+        private static async Task<string> RetryRequest(Func<Task<string>> retryBlock)
+        {
+#if MACOS
```

Comment:
Doesn't this need to be a runtime check? This is packaged by TFM, not by OS.

Reply:
Yeah, you are correct. It depends on runtime.